addon/components/frost-list.js

```diff
@@ -155,6 +172,11 @@ export default Component.extend({
       })
     }
 
+    const itemTypeKey = this.get('itemTypeKey')
```

Comment:
What about the reverse? if `itemTypeKeys` is defined but not `itemTypeKey`?

Reply:
Do you mean if `itemKey` is defined but not `itemTypeKey`?